Src/WitsmlExplorer.Api/Services/LogObjectService.cs

```diff
@@ -145,11 +147,6 @@ public async Task<LogData> ReadLogData(string wellUid, string wellboreUid, strin
             Index startIndex = Index.Start(log, start);
             Index endIndex = Index.End(log, end);
 
-            if (startIndex > endIndex)
```

Comment:
I suggest bringing back this return statement with the following condition to avoid sending out of bounds queries: `if((!log.IsDecreasing() && startIndex > endIndex) || (log.IsDecreasing() && startIndex < endIndex))`.